client/src/App.js

```diff
@@ -2,7 +2,6 @@ import React from 'react';
 import Footer from './component//Footer/Footer.jsx';
 import Main from './component/Main/Main.jsx'
 import NavigationBar from './component/Header/Navigation-Bar.jsx'
-
```

Comment:
Proper Readability includes having a whitespace gap between imports and the class and in between the class and the export. As well as each method in the class with another method